explore/src/main/scala/explore/model/AppContext.scala

```diff
@@ -53,6 +55,27 @@ case class AppContext[F[_]](
   def replacePage(appTab: AppTab, programId: Program.Id, focused: Focused): Callback =
     setPageVia(appTab, programId, focused, SetRouteVia.HistoryReplace)
 
+  def routingLink(
```

Comment:
Good idea! There are a number of places where this could be used.

Reply:
I see you have already refactored to use it.

Reply:
I refactored some of the areas, but I probably missed some.